Code/GraphMol/MolPickler.cpp

```diff
@@ -465,14 +465,15 @@ void pickleQuery(std::ostream &ss, const Query<int, T const *, true> *query) {
         streamWrite(ss, MolPickler::QUERY_VALUE, pval);
       } break;
       case 6: {
-        auto v = boost::get<std::tuple<MolPickler::Tags, Dict::Pair, double>>(
+        auto v = boost::get<std::tuple<MolPickler::Tags, PairHolder, double>>(
```

Comment:
this is copying the tuple. I think you want a reference.

Reply:
Good catch